docs/content/install.md

```diff
@@ -136,6 +136,11 @@ from the rclone image.
   reside on the host with a non-root UID:GID, you need to pass these on the container
   start command line.
 
+- If you want to access the RC interface (either via the API or the Web UI), it is
+  required to set the `--rc-addr` to `0.0.0.0:5572` in order to connect to it from outside
```

Comment:
Using `:5572` instead of `0.0.0.0:5572` should work here - what do you think about changing it?

Reply:
Will do.

While writing it, I thought of the security implications of keeping the port open for everyone if the docker network is set to host instead of a bridge. Should I also add a line warning against that or is that too much information for the install section?

Reply:
I think a sentence here would be appropriate, yes.